addons/website/static/src/components/autocomplete_with_pages/url_autocomplete.js

```diff
@@ -0,0 +1,102 @@
+/** @odoo-module **/
+
+import { Component } from "@odoo/owl";
+import { useService } from "@web/core/utils/hooks";
+import { AutoCompleteWithPages } from "@website/components/autocomplete_with_pages/autocomplete_with_pages";
+
+export class UrlAutoComplete extends Component {
```

Comment:
Randomly commenting here:
![image](https://github.com/odoo/odoo/assets/30048408/31585658-b5ba-4aad-b9fd-2c32afe3efd5)

The dropdown should be below the input, as before:
![image](https://github.com/odoo/odoo/assets/30048408/de34a648-81fa-4ed1-9e9b-89c896ddb93b)


Even if it seems like we had a bug where the dropdown went offscreen (without possibility to scroll), which should be fixed indeed